b2luigi/batch/cache.py

```diff
@@ -16,7 +16,7 @@ class BatchJobStatusCache(abc.ABC, TTLCache):
     which are not started by this b2luigi instance) does not matter.
     """
     def __init__(self):
-        super().__init__(maxsize=1000, ttl=20)
+        super().__init__(maxsize=1000, ttl=120)
```

Comment:
I'm fine with increasing the cache lifetime, I don't have any strong opinions on that. But I would be happy if you could give an explanation of the reasoning why you increased it. Do you think requesting the job status to often might cause the crash? Or is it just so that the TTL is longer than the retry delay?